pygeoapi/process/manager/base.py

```diff
@@ -27,9 +27,10 @@
 #
 # =================================================================
 
-from datetime import datetime
```

Comment:
Can we keep the order of imports:

- standard library (alphabetical order)
- blank line
- 3rd party imports (PyPI)  (alphabetical order)
- blank line
- local imports (alphabetical order)

Reply:
I thought this was in line with keeping the order? 

I usually put `from` imports below straight ones because it is also how [isort](https://pycqa.github.io/isort/docs/configuration/custom_sections_and_ordering.html#controlling-how-isort-sections-from-imports) and other tools do it, and because it is sorted alphabetically. Anyway, no biggie I'll switch it up :wink: